docs/install/releases.md

```diff
@@ -8,8 +8,8 @@ their infrastructure on a staging environment before upgrading a production
 deployment.
 
 We support two release channels:
-[mainline](https://github.com/coder/coder/2.10.0) for the edge version of Coder
-and [stable](https://github.com/coder/coder/releases/latest) for those with
+[mainline](https://github.com/coder/coder/releases/tag/v2.10.1) for the edge version 
```

Comment:
I think we should not call tha `mainline` the edge version. 
For edge we have `ghcr.io/coder/coder-preview:main` that always points to the `main` branch. 

cc: @stirby

Reply:
it is as close to "bleeding edge" has the points mentioned later ie :

**Bleeding Edge: ** This term is often used to describe the very latest version of a software, which includes the most recent changes and features. It's called "bleeding edge" because it's so new that it may not have been thoroughly tested and could have bugs or stability issues. It's often used by developers or early adopters who want to use the latest features and are willing to accept the risk of potential issues.